torch/testing/_internal/distributed/rpc/dist_examples/parameter_server.py

```diff
@@ -0,0 +1,108 @@
+# https://github.com/pytorch/examples/blob/master/distributed/rpc/batch/parameter_server.py
```

Comment:
You might need to add an empty `__init__.py` so that contents in this folder can be found.

Reply:
Let's rename this folder just `examples`, as there is already a `distributed` in the path.